models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -96,6 +96,7 @@ FROM
     ("dac-davinci-coin", "ethereum", "DAC", "0xaad54c9f27b876d2538455dda69207279ff673a5", 18),
     ("dai-dai", "ethereum", "DAI", "0x6b175474e89094c44da98b954eedeac495271d0f", 18),
     ("data-streamr-datacoin", "ethereum", "DATA", "0x0cf0ee63788a0849fe5297f3407f701e122cc023", 18),
+    ("db-dragon-blood", "ethereum", "DB", "0x5423a6a06dd64480ad17c46e42b95827da2719d9", 18),
```

Comment:
API: https://api.coinpaprika.com/v1/coins/db-dragon-blood
From the api we can see that this token may be inactive for now.
Maybe we can add this token for historical price data.

Reply:
The swap on Uniswap is not too active atm. We anticipate this will change once people can claim their airdrops in a couple of weeks. We want to make sure we can publish the price data before then. 

Here is our Dune site:
https://dune.com/energi/dragondegens

Thanks for your help.